### ISSUE TITLE
K23 Gnome beta: Korora logo missing in Gnome control center > details > overview

### ISSUE THREAD

**Issue by exif** [kororaproject/kp-korora-logos#2]
In my testing of the K23 Gnome beta ISO, the "Overview" tab of the "Details" menu category in the Gnome control center shows a tiny question mark icon instead of the Korora logo or other appropriate branding. This occured for me whether in a live system or after installing to bare metal.

**Comment by benjaminvatterj** [comment]
Hi, just moved to K23, no longer beta and this is still missing.

**Comment by csmart** [comment]
Thanks, I expect this is the same as kp-korora-settings-gnome/issues/6 but I don't know what icon is meant to be there.

**Comment by benjaminvatterj** [comment]
Yes we are talking about the same thing. I guess that we need something like /usr/share/icons/korora/48x48/apps/korora-welcome.svg (in K23 Gnome)

**Comment by csmart** [comment]
@benjaminvatterj thanks, but I don't know where about is looking for an icon
